### PR TITLE
Bug 1292681 – XCUITests fail on iPad

### PR DESCRIPTION
**Pull request by varkor**
This fixes various issues with the existing XCUITests on iPad. A couple
of the fixes are a bit hacky, though. XCUITest still seems quite
immature as a testing platform.

### CODE REVIEW

File: XCUITests/BaseTestCase.swift [hunk 1]
```diff
@@ -8,8 +8,9 @@
 
 import XCTest
 
-
 class BaseTestCase: XCTestCase {
+    
+    private var pageLoaded: XCTestExpectation?
```
Comment (farhanpatel) [inline XCUITests/BaseTestCase.swift]: what is this used for?

**Comment by farhanpatel** [comment]
I left the wrong license in the BaseTestCase if you could fix that it would be much appreciated :)

File: XCUITests/BaseTestCase.swift [hunk 2]
```diff
@@ -8,7 +8,6 @@
```
Comment (thebnich) [inline XCUITests/BaseTestCase.swift]: Please fix the license while here.

**Comment by farhanpatel** [comment]
Make sure to squash before merge 😄

**Pull request merged**